cmd/server/main.go

```diff
@@ -39,6 +41,13 @@ func main() {
 
 	logger = logging.CreateLogger(opts.LogEncoding, opts.LogLevel)
 
+	shortGitCommit := GitCommit
+	if len(shortGitCommit) >= 7 {
+		shortGitCommit = shortGitCommit[:7]
+	}
+	clientVersion := "example-notifications-server-go/" + shortGitCommit
+	appVersion := clientVersion
```

Comment:
I don't feel great about using the same value for client and app versions, but don't have a better idea until we publish a go client somewhere

Reply:
One option is to treat the proto repo version as the go client version. The generated code from the repo is the client.